core/modules/system/system.install

```diff
@@ -40,6 +40,22 @@ function system_requirements($phase) {
         'weight' => -9,
       );
     }
+
+    // Report the configuration storage being used.
+    $config_storage_active = config_get_config_storage('active');
+    $config_storage_staging = config_get_config_storage('staging');
+    $backend = array(
+      '@backend' => is_a($config_storage_active, 'ConfigFileStorage') ? t('files') : t('database'),
```

Comment:
If we run this through t() and then also run it through t() in `t('Active configuration: @backend', $backend)` it's going to end up double escaped. 

Can we do `t('Active configuration: !backend', ...),` instead?

Reply:
Of course. Good catch 👍🏼